src/Input/ToggleSwitch/Styles.ts

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { ToggleBallProps, ToggleProps } from './ToggleSwitch.interface';
+
+const ToggleSwitchWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const Toggle = styled.div<ToggleProps>`
```

Comment:
In terms of html semantics, it seems the conventional element for a Toggle component is a `button` instead of a `div`. You will have to add some css to override the default button styles but I think that's an acceptable trade-off for having semantically accurate html. 

One example I found is this Toggle component found here https://scottaohara.github.io/aria-switch-control/
![image](https://user-images.githubusercontent.com/35050056/68006629-3a10c080-fcb4-11e9-8e83-b09c61bb5070.png)